apps/demos/Demos/DropDownButton/Overview/jQuery/index.js

```diff
@@ -20,11 +20,36 @@ $(() => {
     onItemClick(e) {
       DevExpress.ui.notify(e.itemData.name, 'success', 600);
     },
-    text: 'Sandra Johnson',
-    icon: '../../../../images/gym/coach-woman.png',
+    text: 'Olivia Peyton',
     displayExpr: 'name',
     keyExpr: 'id',
     useSelectMode: false,
+    template(data, element) {
+      const $imageWrapper = $("<div>")
```

Comment:
```suggestion
      const $imageWrapper = $("<div>")
```

Please rename all instances of `imageWrapper` to `imageContainer`.